libs/saksbehandling-common/src/main/kotlin/Retry.kt

```diff
@@ -12,6 +12,16 @@ sealed class RetryResult<T> {
     }
 }
 
+suspend fun <T> retryOgPakkUt(
```

Comment:
Hva er det man pakker ut egentlig?

Reply:
Det er vel enkelt sagt let-blokka - `it.content` og `it.samlaExceptions()` er utpakking frå Result-objektet